Gemfile

```diff
@@ -39,12 +39,13 @@ gem "govuk_message_queue_consumer", "~> 2.1.0"
 gem "govuk-lint", "~> 1.0.0"
 
 group :test do
+  gem "minitest", "4.7.5"
+  gem "minitest-colorize"
```

Comment:
Can you use pessimistic versioning for these?

https://robots.thoughtbot.com/rubys-pessimistic-operator